### PR TITLE
#162342685 Configure ESlint to review PRs

### PR DESCRIPTION
**Pull request by cdvx**
#### What does this PR do?
Configure ESlint to review PRs

#### Description of Task to be completed?
Makes sure ESlint is configured to check for style violations

#### How should this be manually tested?
Raise a PR and ESlint will make a review based on es6 coding style conventions.

#### Any background context you want to provide?
To test this out, one has to download and install `nodejs` locally (on the development machine)
install yarn `brew install yarn`
clone the repository
`git clone https://github.com/andela/ah-frontend-thor.git`

Then install the dependencies with
`yarn install` or `yarn`

#### What are the relevant pivotal tracker stories?
#162342685

### CODE REVIEW

**Review by joshNic — APPROVED** [review]

**Review by johnkegz — APPROVED** [review]
LGTM

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# version: "2"
+# engines:
+# 	eslint:
+# 		enabled: true
```
Comment (shakirandagire) [inline .codeclimate.yml]: Could you remove this commented out code

**Comment by shakirandagire** [comment]
@cdvx Please fix the merge conflicts on your branch

File: .codeclimate.yml (hunk 1 above)
Comment (cdvx) [reply .codeclimate.yml]: @shakirandagire I've removed the commented code.

**Pull request merged**